src/api/config/options.yml.example

```diff
@@ -203,6 +203,10 @@ default: &default
   #   description: We are the best at ABC
   #   icon: sponsor_abc
   #   url: https://www.example.com
+  #
+  # Set the waiting time until the published repositories can be removed.
+  # The default value is 2 days (172800 seconds).
+  removal_delay: 172800
```

Comment:
Great!

Reply:
The name and description of this option sound scary. That this is only about accepting maintenance release requests is far from obvious

Reply:
Maybe `published_repositories_cleanup_job_delay`?

Reply:
I would claim in about all OBS installations, this option is not relevant as no maintenance release requests are seen. But every installation has published repositories *not* cleaned up by this setting.